src/pysimfrac/src/methods/to_2D.py

```diff
@@ -27,7 +25,6 @@ def generate_2D(self, ind=None, mean_aperture=28):
     
     self.top = np.tile( self.top[ind], (self.top.shape[1], 1))
     self.bottom = np.tile( self.bottom[ind], (self.top.shape[1], 1))
-    self.set_mean_aperture(mean_aperture) 
```

Comment:
I think this should be self.set_mean_aperture() as mean_aperture is not defined here other than self.mean_aperture

Reply:
Thank you, I understand, I modified it